config/config.go

```diff
@@ -6,36 +6,41 @@ import (
 	yaml "gopkg.in/yaml.v1"
 )
 
-// Homebrew contains the brew section
-type Homebrew struct {
+// HomebrewConfig contains the brew section
```

Comment:
I think the API would be simpler when going with the [convention](https://golang.org/doc/effective_go.html#package-names) of stripping the `Config` suffix for all types.
Then you could use them from other packages like `config.Homebrew` or `config.Project`.